width_fov: 1.23220245191      # 70.6*M_PI/180
height_fov: 1.0471975512      # 60.0*M_PI/180
synchronize_frames: false
